Rakefile

```diff
@@ -0,0 +1,11 @@
+$:.unshift File.expand_path("../lib", __FILE__)
```

Comment:
Is there a way to leverage Bundler to add this directory to the load path? Manually modifying `$:` feels a little sketchy.